website/api_versioned_docs/version-0.18/bonsai/eth-examples.md

```diff
@@ -11,7 +11,7 @@ While all of the [zkVM examples][zkvm-examples] can be run on Bonsai by [configu
 This [example app][governance-example] uses Bonsai as an Ethereum coprocessor. The protocol, based on the OpenZeppelin [Governor smart contract standard], batches signature verifications off-chain for a DAO governance vote. The end result is that in [~160 lines of Rust][signature-aggregation], a gas savings of 66% is achieved with significant room for optimizations.
 
 [zkvm-examples]: /api/zkvm/examples
-[remote-proving]: /api/zkvm/quickstart#remote-proving
+[remote-proving]: /api/generating-proofs/remote-proving
```

Comment:
continuing the conversation about absolute vs. relative paths -- this works for now but will break if we ever do a docs re-organization. 

pointing to the associated reference on the `0.18` page would be better. I think that link is:
```suggestion
[remote-proving]: ../zkvm/quickstart#remote-proving
```

Reply:
adding on to that thought-- if we had been using a relative link in the first place, the content migration from `zkvm/quickstart#remote-proving` to `generating-proofs/remote-proving` wouldn't have broken our link